app/presenters/policies_list_presenter.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class PoliciesListPresenter
+  def initialize(policies)
+    @policies = policies
+  end
+
+  def get_registry
```

Comment:
We never name in ruby a method like `get_...`, we just put what it returns or does directly.

Reply:
Busted! 🤣